config/doid.yml

```diff
@@ -3,26 +3,33 @@
 idspace: DOID
 base_url: /obo/doid
 
-base_redirect: https://github.com/DiseaseOntology/HumanDiseaseOntology/tree/master/src/ontology/
+base_redirect: https://github.com/DiseaseOntology/HumanDiseaseOntology
 
 products:
 - doid.owl: https://raw.githubusercontent.com/DiseaseOntology/HumanDiseaseOntology/master/src/ontology/doid.owl
 - doid.obo: https://raw.githubusercontent.com/DiseaseOntology/HumanDiseaseOntology/master/src/ontology/doid.obo
+- doid.json: https://raw.githubusercontent.com/DiseaseOntology/HumanDiseaseOntology/master/src/ontology/doid.json
 
 term_browser: ontobee
 example_terms:
 - DOID_4
 
 entries:
 
+- exact: /
```

Comment:
`exact` can't have tests. Should this be `prefix`? If so, it should be the last entry, not the first, since entries are checked in the order they appear.

Reply:
Whoops, yes, this should be `prefix`. Fixed.